integration_tests/ebpf_test.go

```diff
@@ -132,6 +179,9 @@ func setupDevice(path *string, dev *uint32) (func() error, error) {
 }
 
 func TestBasic(t *testing.T) {
+	if os.Getenv("ONLY_APPARMOR") == "true" {
```

Comment:
nit: is the plan to keep `ONLY_APPARMOR` or was it for ease of development?

Reply:
i couldn't run the test suite at all. apparently it needs a `disk.img` that i don't know how to craft _just right_ cc: @cbrewster